src/Utility/Twig/Loader.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SilverStripe\Cow\Utility\Twig;
+
+use Twig\Loader\FilesystemLoader;
+use SilverStripe\Cow\Application;
+
+class Loader extends FilesystemLoader
+{
+    /**
+     * Instance of Cow Application
+     *
+     * @var Application
+     */
+    private $application;
+
+    public function __construct(Application $cow)
```

Comment:
I was aiming to encapsulate the business logic of the loader within loader. Since it uses Application interface as a dependency, it has access to anything it may want.
Otherwise, we'd have to put the Twig Loader initialization logic into the Application (which is too high level to care about twig init).

Reply:
> Loader::$application doesn't seem to be used either?

oh, true. I'll remove

Reply:
done